taxonomies/categories.txt

```diff
@@ -18191,7 +18191,8 @@ pnns_group_2:en:Dairy desserts
 #Do not confound with the fruit preserve called jelly
 <en:Desserts
 en:Jellies (dessert)
```

Comment:
We can't have parenthesis in the category names, "Jellies (dessert)" will be turned in to "Jellies, dessert", and it will make dessert a synonym of jellies.

Reply:
(the issue was here before, I just saw it because of the new translation)

Reply:
I removed the parenthesis for the English synonym. Can I let them in the other languages?

Reply:
No, we need to remove them in all languages, otherwise we have the same synonyms issue.

Reply:
I made the changes! Should I also change the categories "fr:Valençay (vin)" and "Valençay (fromage)" to remove the parenthesis?

Reply:
Yes please.

Reply:
I corrected all the parenthesis I found except some in pl and ru

Reply:
Thank you!